### ISSUE TITLE
Add option to prevent horizontal scrolling if unnecessary

### ISSUE THREAD

**Issue by opeik** [qvacua/vimr#717]
Scrolling with the trackpad works well until it starts moving horizontally by accident, which is very disorientating. An option to prevent horizontal scrolling if there's no need to scroll would be great.

**Comment by kswope** [comment]
Yes please fix this.  It seems to want to scroll until it can only display the farthest right characters that is *current* on the screen, which makes it wackier when you scroll right and vertically and the farthest right character changes.

**Comment by kswope** [comment]
Yes please fix this.  It seems to want to scroll until it can only display the farthest right characters that is *current* on the screen, which makes it wackier when you scroll right and vertically and the farthest right character changes.

**Comment by kswope** [comment]
I found a workaround for this.  It works for me because I don't let my line length get to big.  But it might cause a mess in files with long lines.
```
set wrap
```
